### DISCUSSION TITLE
Actividad Docker - Grupo 12

### DISCUSSION THREAD

**Discussion by NicoPavoni** [frlp-utn-ingsoft/catedra#111] [Show and tell]
https://github.com/NicoPavoni/vetsoft

**Comment by rodriguezemautn** [comment]
Correcciones:
- el archivo env-example no lleva . (punto)
- los endpoint dan error, no se migra la base de datos(verificar orden del CMD en el Dockerfile o utilizar un script.sh en un ENTRYPOINT) 
- README.md no sugiere como acceder a la app

Puntos aprobados:
* construye la imagen
* buen tamaño de imagen
* accede a la app (pero los endpoints dan error)
* continua ejecuncion en segundo plano (-d)

Salida de comandos:
 docker ps  -a
$ cabb90768108   vetsoft-app1.0   "python manage.py ru…"   4 seconds ago   Up 3 seconds   0.0.0.0:8000->8000/tcp, :::8000->8000/tcp   stupefied_turing
$ docker images -a
vetsoft-app1.0   latest    d4ecefdd753b   55 seconds ago   363MB

**Reply by NicoPavoni** [reply]
Hola @rodriguezemautn ya están subidos los cambios con las correcciones. Gracias!
